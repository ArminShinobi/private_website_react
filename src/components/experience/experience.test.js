import renderer from 'react-test-renderer';
import Experience from './experience';

it('Verify Experience Snapshot', () => {
  const tree = renderer.create(<Experience />).toJSON();
  expect(tree).toMatchSnapshot();
});
