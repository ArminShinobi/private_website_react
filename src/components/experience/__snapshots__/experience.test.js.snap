// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify Experience Snapshot 1`] = `
<div
  className="background-img"
  style={
    Object {
      "backgroundImage": "url(water_and_fire_background.jpg)",
    }
  }
>
  <div
    className="d-flex flex-column min-vh-100 justify-content-center align-items-center"
  >
    <div
      className="col-lg-12 text-center"
    >
      <h2
        style={
          Object {
            "marginTop": "5rem",
          }
        }
      >
        Experience
      </h2>
      <h6>
        Over the course of time, I have learned and worked with many programming languages and tools.
        <br />
         I also earned some certificates
      </h6>
    </div>
    <div
      className="col-lg-4 m-3"
    >
      <h2>
        Programming languages
      </h2>
      <ul>
        <li>
          Python
        </li>
        <li>
          Java
        </li>
        <li>
          JavaScript/Typescript
        </li>
        <li>
          PHP
        </li>
        <li>
          C#
        </li>
      </ul>
      <h2>
        Tools
      </h2>
      <ul>
        <li>
          Git in combination with GitHub/Stash
        </li>
        <li>
          MySQL in combination with MariaDB
        </li>
        <li>
          Bootstrap
        </li>
        <li>
          React
        </li>
        <li>
          AWS
        </li>
        <li>
          Docker
        </li>
        <li>
          Jira
        </li>
        <li>
          Robot Framework
        </li>
      </ul>
      <h2>
        Certificates
      </h2>
      <ul>
        <li>
          Crash Course on Python
        </li>
        <li>
          Docker for absolute beginners
        </li>
        <li>
          Neural Networks and Deep Learning
        </li>
        <li>
          Improving Deep Neural Networks: Hyperparameter Tuning, Regularization and Optimization
        </li>
      </ul>
    </div>
  </div>
</div>
`;
